CHANGELOG.md

```diff
@@ -9,12 +9,18 @@
 **Bugfixes**
   * Fixed an issue where requiring just the local v4 driver wouldn't work due to not `require`ing selenium capabilities
 
+## <sub>v1.2</sub>
```

Comment:
don't place a version cut yet. As we have yet to decide on one (Also it's likely to be a breaking one).